### PR TITLE
SONARPY-941 Make sure there are no conflicting symbols having the sam…

### PR DESCRIPTION
**Pull request by andrea-guarino-sonarsource**
…e name across Python versions

### CODE REVIEW

File: python-frontend/src/test/java/org/sonar/python/types/TypeShedTest.java [hunk 1]
```diff
@@ -46,6 +46,7 @@
 import org.sonar.python.types.protobuf.SymbolsProtos;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
```
Comment (guillaume-dequenne-sonarsource) [inline python-frontend/src/test/java/org/sonar/python/types/TypeShedTest.java:49]: It seems this import (as well as `org.junit.Ignore`on line 31) is unused.

**Review by guillaume-dequenne-sonarsource — APPROVED** [review]
LGTM.
To be clear, does that mean this issue was actually fixed in a prior PR and this one is just here to update the tests?

File: python-frontend/src/test/java/org/sonar/python/types/TypeShedTest.java (hunk 1 above)
Comment (andrea-guarino-sonarsource) [reply python-frontend/src/test/java/org/sonar/python/types/TypeShedTest.java:49]: thanks, I'll remove it!

**Comment by andrea-guarino-sonarsource** [comment]
> 

Yes exactly, I'll add a comment in ticket to clarify the actual fix has been done in another PR

**Comment by andrea-guarino-sonarsource** [comment]
Thinking a bit more about the solution we took in `TypeShed. disambiguateSymbolsWithSameName`, I think it makes sense to have a debug log when we disambiguate with a Python 3 symbol. This might be helpful during an investigation. I'll add this log in a new  commit, let me know if you think it's not a good idea

**Pull request merged**